runtime/testing-basilisk/src/lib.rs

```diff
@@ -919,6 +946,9 @@ construct_runtime!(
 		Marketplace: pallet_marketplace = 109,
 		TransactionPause: pallet_transaction_pause = 110,
 
+		LiquidityMining: pallet_xyk_liquidity_mining = 111,
```

Comment:
nitpick: won't we have a stable swap liquidity mining? then this should be `XYKLiqduityMining` or similar

Reply:
yes, I was thinking about renaming this but FE is already using in on testnet and my understanding was that we want to deploy it directly to rococo without testnet fist.